Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -207,11 +204,11 @@ SHADER* ProgramShaderCache::SetShader(DSTALPHA_MODE dstAlphaMode, u32 primitive_
 	last_entry = &newentry;
 	newentry.in_cache = 0;
 
-	ShaderCode vcode = GenerateVertexShaderCode(API_OPENGL);
-	ShaderCode pcode = GeneratePixelShaderCode(dstAlphaMode, API_OPENGL);
+	ShaderCode vcode = GenerateVertexShaderCode(API_OPENGL, uid.vuid.GetUidData());
+	ShaderCode pcode = GeneratePixelShaderCode(dstAlphaMode, API_OPENGL, uid.puid.GetUidData());
 	ShaderCode gcode;
 	if (g_ActiveConfig.backend_info.bSupportsGeometryShaders && !uid.guid.GetUidData()->IsPassthrough())
-		gcode = GenerateGeometryShaderCode(primitive_type, API_OPENGL);
+		gcode = GenerateGeometryShaderCode(primitive_type, API_OPENGL, uid.guid.GetUidData());
 
 	if (g_ActiveConfig.bEnableShaderDebugging)
```

Comment:
At least this usage has no effect any more. It stored the source which is never read again.